doc/examples/data/plot_3d.py

```diff
@@ -0,0 +1,25 @@
+"""
+==========================================
+datasets with 3 or more spatial dimensions
```

Comment:
```suggestion
Datasets with 3 or more spatial dimensions
```
(all gallery examples have capitalized titles)